apps/functions/deadline-submissions/deadline-submission-command/event-client.js

```diff
@@ -0,0 +1,25 @@
+import { getEventClient, EventType } from '@pins/event-client';
+import config from './config';
+
+const { serviceBusHost, serviceBusTopic } = config;
+
+/**
+ *
+ * @param {import('@azure/functions').Context} context
+ * @param {{ deadline: string, submissionType: string, blobGuid: string, documentName: string }} _
+ * */
+async function sendEvent(context, { deadline, submissionType, blobGuid, documentName }) {
+	const eventClient = getEventClient(true, context.log, serviceBusHost);
+
+	const event = {
+		status: 'FAILURE',
```

Comment:
Is this always meant to be failure here?

Reply:
Yeah if this runs it must be a failure. We don't want to publish SUCCESS until the virus check function has run and not found anything. So the happy path for this function would have it not publishing anything, but if it fails it publishes a FAILURE event.

Reply:
In that case perhaps the function should be named differently? It doesn't seem obvious this is sending a failure event specifically.